serverless-workflow-examples/serverless-workflow-data-index-quarkus/README.md

```diff
@@ -205,4 +207,4 @@ After a while (note that to you need give time for event to be consumed)  you sh
 
 ### Query process details in Data Index
 
-Data Index GraphQL UI is available at http://localhost:8080/q/graphql-ui/
\ No newline at end of file
+Data Index GraphQL UI is available at http://localhost:8180/graphiql/
```

Comment:
Is this correct? I am getting 404, what works is this one - `http://localhost:8180/q/graphql-ui/` I guess on midstream we miss a change?

Reply:
I think http://localhost:8180/q/graphql-ui/ works in dev mode,  and the http://localhost:8180/graphiql/ is the endpoint when it's started with the provided docker-compose